src/pymatgen/alchemy/materials.py

```diff
@@ -369,7 +369,7 @@ def to_snl(self, authors: list[str], **kwargs) -> StructureNL:
             history += [
                 {
                     "name": snl_metadata.pop("name", "pymatgen"),
-                    "url": snl_metadata.pop("url", "http://pypi.python.org/pypi/pymatgen"),
+                    "url": snl_metadata.pop("url", "https://pypi.python.org/pypi/pymatgen"),
```

Comment:
This looks like a breaking change too, the metadata (default value of `url` would have been changed):
https://github.com/materialsproject/pymatgen/blob/46d843c07b7dcfd1ab25ab41a01cfa60f5ce123d/src/pymatgen/alchemy/materials.py#L367-L375